### PR TITLE
Update Book.cs

### PR DESCRIPTION
**Pull request by ayesha1906**

### CODE REVIEW

File: Linq/Linq/Book.cs [hunk 1]
```diff
@@ -4,6 +4,5 @@ public class Book
     {
         public string Title { get; set; }
         public float Price { get; set; }
-
```
Comment (ayesha1906) [inline Linq/Linq/Book.cs:7]: Why did u remove this line?

Comment (greymonk) [inline Linq/Linq/Book.cs:7]: why this has been removed?

**Review by greymonk — CHANGES_REQUESTED** [review]

**Review by ayesha1906 — COMMENTED** [review]
unnecessary space

File: Linq/Linq/Book.cs (hunk 1 above)
Comment (ayesha1906) [reply Linq/Linq/Book.cs:7]: unnecessary space

Comment (greymonk) [reply Linq/Linq/Book.cs:7]: cool

**Review by greymonk — APPROVED** [review]

**Pull request merged**